src/goRunTestCodelens.ts

```diff
@@ -30,7 +30,7 @@ export class GoRunTestCodeLensProvider extends GoBaseCodeLensProvider {
 			return [];
 		}
 		let config = vscode.workspace.getConfiguration('go', document.uri);
-		let codeLensConfig = config.get('enableCodeLens');
+		let codeLensConfig: { [key: string]: any } = config.get('enableCodeLens');
```

Comment:
uhh, i guess there is no type for these configs ? :(

Reply:
Nope... the object itself follows the structure go.enableCodeLens object at package.json (and configurable by the user), but it doesn't have an explicit type to set

Reply:
yea that would be a pain, there has to be a way to type those without manually mantaining them consistent, ill go into it at one point, its fine for the scope if this PR.